### PR TITLE
Convert TestMtlsStrictK8sCA into multicluster support

### PR DESCRIPTION
**Pull request by jtrbs**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[X] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[X] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Comment by jtrbs** [comment]
Test passed at https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/28720/integ-security-multicluster-tests_istio/1326044741871079424

**Comment by jtrbs** [comment]
Test passed at https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/28720/integ-security-multicluster-tests_istio/1326324562039672832

File: tests/integration/security/util/framework.go [hunk 1]
```diff
@@ -85,6 +82,26 @@ func EchoConfig(name string, ns namespace.Instance, headless bool, annos echo.An
 				Protocol: protocol.GRPC,
 			},
 		},
+		// Workload Ports needed by TestPassThroughFilterChain
+		// The port 8085,8086,8087,8088 will be defined only in the workload and not in the k8s service.
+		WorkloadOnlyPorts: []echo.WorkloadPort{
```
Comment (jtrbs) [inline tests/integration/security/util/framework.go:87]: Hi @stevenctl, this is to address the comment in https://github.com/istio/istio/pull/28502

**Review by stevenctl — APPROVED** [review]

File: tests/integration/security/pass_through_filter_chain_test.go [hunk 2]
```diff
@@ -115,7 +115,7 @@ func TestPassThroughFilterChain(t *testing.T) {
 					},
 
 					// For workload c, there is only authN policy that enables mTLS (Strict).
-					// The request should be denied because the x is always using plain text.
+					// The request should be denied because the e is always using plain text.
```
Comment (JimmyCYJ) [inline tests/integration/security/pass_through_filter_chain_test.go:118]: Is this change intended (replacing x with e)?

**Review by JimmyCYJ — APPROVED** [review]

File: tests/integration/security/pass_through_filter_chain_test.go (hunk 2 above)
Comment (jtrbs) [reply tests/integration/security/pass_through_filter_chain_test.go:118]: Yes
